packages/lab/src/Flow/Sidebar/Sidebar.tsx

```diff
@@ -40,6 +39,8 @@ export interface HvFlowSidebarProps
   title?: string;
   /** Sidebar description. */
   description?: string;
+  /** Flatten sidebar items */
+  flatten?: boolean;
```

Comment:
`defaultGroupProps` should be removed for the `HvFlowSidebarProps` since it's no longer needed